### PR TITLE
[MIGR] Adding l10n_ch_scan_bvr

### PR DESCRIPTION
**Pull request by hurrinico**

### CODE REVIEW

File: l10n_ch_scan_bvr/wizard/scan_bvr.py [hunk 1]
```diff
@@ -85,22 +71,22 @@ def _check_number(self, part_validation):
     def _construct_bvrplus_in_chf(self, bvr_string):
 
             if len(bvr_string) != 43:
-                raise orm.except_orm(
+                raise exceptions.Warning(
                     _('Validation Error'),
```
Comment (tafaRU) [inline l10n_ch_scan_bvr/wizard/scan_bvr.py]: Warning() takes 1 argument (no title)

File: l10n_ch_scan_bvr/wizard/scan_bvr.py [hunk 2]
```diff
@@ -85,22 +71,22 @@ def _check_number(self, part_validation):
...
                     _('BVR CheckSum Error in first part')
                 )
             elif self._check_number(bvr_string[0:2]) != int(bvr_string[2]):
-                raise orm.except_orm(
+                raise exceptions.Warning(
                     _('Validation Error'),
```
Comment (tafaRU) [inline l10n_ch_scan_bvr/wizard/scan_bvr.py]: Same as above

File: l10n_ch_scan_bvr/wizard/scan_bvr.py (hunk 1 above)
Comment (tafaRU) [inline l10n_ch_scan_bvr/wizard/scan_bvr.py]: @hurrinico please fix this in all occurences. Thank you in advance.

File: l10n_ch_scan_bvr/partner.py [hunk 3]
```diff
@@ -19,17 +19,15 @@
...
-    _columns = {
-        'supplier_invoice_default_product': fields.many2one(
-            'product.product',
-            'Default product supplier invoice',
-            help="Use by the scan BVR wizard, if completed, it'll generate "
-                 "a line with the proper amount and this specified product"
-        ),
-    }
+    supplier_invoice_default_product = fields.Many2one(
+        'product.product',
```
Comment (tafaRU) [inline l10n_ch_scan_bvr/partner.py]: Please add `comodel_name=`

File: l10n_ch_scan_bvr/__openerp__.py [hunk 4]
```diff
@@ -44,9 +44,11 @@
...
  "depends": ["l10n_ch",
-             "l10n_ch_payment_slip"],
+             "l10n_ch_payment_slip",
+             "account"],
  "data": ["wizard/scan_bvr_view.xml",
           "partner_view.xml",
           "bank_view.xml"],
- 'installable': False
+ 'installable': True,
+ 'application': True,
```
Comment (eLBati) [inline l10n_ch_scan_bvr/__openerp__.py]: why `application`?

**Comment by eLBati** [comment]
@hurrinico see Travis: tests are broken.
Also, maybe you need to rebase to avoid conflicts

**Comment by nbessi** [comment]
@hurrinico can you please rebase

**Comment by nbessi** [comment]
@vrenaville  can you please review

**Comment by yvaucher** [comment]
@hurrinico What is the state of this PR? It seems the tests are still broken. Any chance you will fix them?

**Comment by hurrinico** [comment]
@yvaucher i'll work on it ASAP

**Comment by yvaucher** [comment]
@hurrinico Have you seen #163 your was there first and #163 needs a lot of cleaning as based on 6.1 version. Maybe @open-net-sarl could make a PR to improve this one?

**Comment by yvaucher** [comment]
Thanks for the quick reply, just ping me when you come up with something.

**Comment by tafaRU** [comment]
@yvaucher, what do you suggest to finalize the migration? Can we continue on this branch?

Thank you in advance.

**Comment by yvaucher** [comment]
@tafaRU As @open-net-sarl didn't want to invest more resources on their PR, yes, let's keep this PR

**Comment by tafaRU** [comment]
Superseeded by https://github.com/OCA/l10n-switzerland/pull/169

**Comment by yvaucher** [comment]
Thanks @tafaRU Thus I close this one

**Pull request closed without merging**